### PR TITLE
Refactor components and add dynamic data

### PR DESCRIPTION
**Pull request by justin-phxm**
image

Now fetches alumni from Contenful. Order of alumni has been reversed.

### CODE REVIEW

**Comment by burtonjong** [comment]
Looks good to me - The encryption works as intended as well

Just be sure to remove your console.log before merging into main
image

File: pages/who-we-are/index.js [hunk 1]
```diff
@@ -16,17 +22,17 @@ const FAQDropdown = ({ prompt, content }) => {
           <>
             <Disclosure.Button className="items-center drop-shadow-md flex w-full justify-between rounded-lg bg-[#7055FD] px-8 py-4 text-left focus:outline-none focus-visible:ring focus-visible:ring-purple-500 focus-visible:ring-opacity-75">
               <span className="text-white italic text-xl font-medium">&quot;{prompt}&quot;</span>
-              <FaAngleDown className={`${open ? "rotate-180 transform" : ""} h-5 w-5 text-white`} />
+              <FaAngleDown className={`${open && 'rotate-180 transform'} h-5 w-5 text-white`} />
```
Comment (topanb11) [inline pages/who-we-are/index.js:26]: nice refactor 💯

File: components/AlumniCarousel.jsx [hunk 2]
```diff
@@ -37,11 +56,12 @@ const AlumniTile = ({ name, file, position, prevRole, testimonial, linkedin }) =
   );
 };
 
-const AlumniCarousel = () => {
+const AlumniCarousel = ({ alumni }) => {
+  console.log(alumni);
```
Comment (topanb11) [inline components/AlumniCarousel.jsx]: delete this `console.log`

File: components/AlumniCarousel.jsx [hunk 3]
```diff
@@ -37,11 +56,12 @@ const AlumniTile = ({ name, file, position, prevRole, testimonial, linkedin }) =
...
   return (
     <div className={CAROUSEL_CONTAINER}>
       <Carousel>
-        {ALUMNI.map((alumni) => (
+        {alumni.map((alumni) => (
```
Comment (topanb11) [inline components/AlumniCarousel.jsx:63]: when you map them, could you sort them by the `orderNumber`?

File: components/AlumniCarousel.jsx [hunk 4]
```diff
@@ -1,27 +1,46 @@
...
-const CAROUSEL_CONTAINER = "text-white rounded-lg w-[20rem] h-[44rem] shadow-2xl md:w-4/5 swiper-button-white";
+const CAROUSEL_CONTAINER = 'text-white rounded-lg w-[20rem] h-[44rem] shadow-2xl md:w-4/5 swiper-button-white';
 const ALUMNI_TILE =
-  "bg-[#7055FD] w-[20rem] h-[70rem] md:h-[44rem] rounded-lg items-center p-14 lg:p-10 flex flex-col shadow-2xl md:w-full md:flex-row";
-const ALUMNI_PROFILE = "flex flex-col items-center mx-auto text-center md:p-10";
+  'bg-[#7055FD] w-[20rem] h-[70rem] md:h-[44rem] rounded-lg items-center p-14 lg:p-10 flex flex-col shadow-2xl md:w-full md:flex-row';
+const ALUMNI_PROFILE = 'flex flex-col items-center mx-auto text-center md:p-10';
 
-const AlumniTile = ({ name, file, position, prevRole, testimonial, linkedin }) => {
+const keyStr = 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789+/=';
```
Comment (topanb11) [inline components/AlumniCarousel.jsx:14]: question: what's this for

**Review by topanb11 — COMMENTED** [review]
thanks Justin! Looks really good on desktop/mobile and real-time updates work on Contentful. this is a feature we've been wanting to add for a minute so great job! 👏

Just left two small comments and a question

File: components/AlumniCarousel.jsx (hunk 4 above)
Comment (justin-phxm) [reply components/AlumniCarousel.jsx:14]: This is just some code I copied from the NextJS official examples: https://github.com/vercel/next.js/blob/canary/examples/image-component/app/color/page.tsx

Apparently, what it does is it generates a DataURL for a 1x1 pixel image as a placeholder while the image is fetched and loaded. The keyStr contains all the characters in a 64-bit encoding which is used to convert an RGB value to binary data.

**Review by arminsandhu — APPROVED** [review]
Looks good Justin!!! Thank for applying the feedback and refactoring the code 🙌

**Pull request merged**